packages/lib/modules/pool/actions/add-liquidity/modal/StakingOptions.tsx

```diff
@@ -54,32 +62,34 @@ export function StakingOptions() {
             </Button>
           </VStack>
         </Card>
-        <Card position="relative" variant="modalSubSection">
-          <VStack align="left" spacing="md">
-            <Text color="grayText">Aura</Text>
-            <HStack>
-              <Text color="font.primary" fontSize="md" fontWeight="bold">
-                {pool.staking?.aura ? fNum('apr', pool.staking.aura.apr) : 'Not available'}
-              </Text>
-            </HStack>
-            <Flex position="absolute" right={2} top={3}>
-              <Image alt="balancer" height={30} src="/images/protocols/aura.svg" width={30} />
-            </Flex>
-            {pool.staking && pool.staking.aura && (
-              <>
-                <Button onClick={auraDisclosure.onOpen} variant="secondary" w="full">
-                  Learn more
-                </Button>
-                <PartnerRedirectModal
-                  isOpen={auraDisclosure.isOpen}
-                  onClose={auraDisclosure.onClose}
-                  partner={RedirectPartner.Aura}
-                  redirectUrl={getAuraPoolLink(getChainId(chain), pool.staking.aura.auraPoolId)}
-                />
-              </>
-            )}
-          </VStack>
-        </Card>
+        {isBalancerProject() && pool.staking?.aura && (
```

Comment:
I'm a bit worried about conditional rendering like this within the lib folder. I guess somehow we need to split components like this out into sub-project-specific packages. But I realise you're in a rush so we can do this for now.